test/org/sagebionetworks/bridge/services/SendMailViaAmazonServiceConsentTest.java

```diff
@@ -78,25 +77,6 @@ public void setUp() throws Exception {
         studyConsentService = mock(StudyConsentService.class);
         when(studyConsentService.getActiveConsent(any(StudyIdentifier.class))).thenReturn(view);
     }
-    
-    @Test
```

Comment:
This should not happen. Study supportEmail is required and present for all studies, we never want to send these from our default email address.